src/libs/vehicle/ardupilot/ardupilot.ts

```diff
@@ -90,7 +90,7 @@ export abstract class ArduPilotVehicle<Modes> extends Vehicle.AbstractVehicle<Mo
   onMAVLinkMessage = new SignalTyped()
   _flying = false
 
-  protected currentSystemId = 1
```

Comment:
The problem of having this variable public is that someone will be able to overwrite the systemID, this will result in bigger problems. The safest way is to have an method or readonly property that provides such variable.

Reply:
Returned to `protected` and added a getter.